src/components/interface/Section.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { withStyles } from '@material-ui/core/styles';
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+
+import styles from '../styles';
+import EmptyState from './EmptyStates';
+import Loading from './Loading';
+import SectionHeader from './SectionHeader';
+import TableHeader from './TableHeader';
+import TableFilters from './TableFilters';
+
+const Section = ({
+  children,
+  headings,
+  loading,
+  showContent,
+  type
+}) => {
+  const showNewButton = type !== 'adjudication';
```

Comment:
Typically, we should try to name boolean equalities the same as what they represent. I'd almost rather instead of `!==`, we do `const isAdjudication = type === 'adjudication'` and use `showNew={!isAdjudication}`